bin/install-site.sh

```diff
@@ -586,17 +585,28 @@ fi
 DOCKER=false
 INSTALL_DB=true
 INSTALL_POSTFIX=true
+PACKAGE_SUFFIX=
 if [ x"$1" = x"--docker" ]
 then
     DOCKER=true
     INSTALL_DB=false
     DEFAULT_SERVER=true
+    PACKAGE_SUFFIX=docker
     shift
 fi
 
+if [ x"$1" = x"--slim" ]
+then
+  DOCKER=false
```

Comment:
Picky, but spacing isn't consistent :)

Reply:
Oops, my default spacing is 2 but that file is 4 - fixed.